### ISSUE TITLE
Zoom pinch spasming 

### ISSUE THREAD

**Issue by kiliancs** [airbnb/visx#1638]
The zoom example in https://airbnb.io/visx/zoom-i demonstrates the issue.

https://user-images.githubusercontent.com/98786/214480791-32a4b3bf-d9bb-4776-82e8-9e13fde43cc5.mov

I'm seeing this in Firefox and Chrome, but not Safari, while using the pinch gesture using the touchpad.

**Comment by atfera** [comment]
I don't know if it's related, but I have the same issue when using wheel + holding ctrl key, in Firefox and Chrome but not Safari.

**Comment by kiliancs** [comment]
I have noticed that the wheel handler (`handleWheel`) is fired when pinching, as is `handlePinch`. Occasionally, the two events scale in the opposite direction, causing the issue.

<img width="486" alt="Screen Shot 2023-01-25 at 11 30 37 AM" src="https://user-images.githubusercontent.com/98786/214620430-1a59c6b2-f27c-4be7-a154-a67de9aee614.png">

**Comment by kiliancs** [comment]
One thing to try is to ignore wheel events while pinching, similar to how it's done in `onDrag`:

```ts
      onDrag: ({ event, pinching, cancel }) => {
        if (pinching) { // <----
          cancel();
          dragEnd();
        } else if (!(event instanceof KeyboardEvent)) {
          dragMove(event);
        }
      },
```

**Comment by kiliancs** [comment]
My attempt at a fix #1639.
